src/object/C4Object.cpp

```diff
@@ -231,37 +231,70 @@ void C4Object::AssignRemoval(bool exit_contents)
 	{
 		C4RCCreateObj rc;
 		memset(&rc, '\0', sizeof(rc));
-		rc.oei=Number;
-		if (Def && Def->GetName()) strncpy(rc.id, Def->GetName(), 32+1);
-		rc.x=GetX(); rc.y=GetY(); rc.ownr=Owner;
+		rc.oei = Number;
+		if (Def && Def->GetName())
+		{
+			strncpy(rc.id, Def->GetName(), 32+1);
+		}
+		rc.x = GetX();
+		rc.y = GetY();
+		rc.ownr = Owner;
 		AddDbgRec(RCT_DsObj, &rc, sizeof(rc));
 	}
+
+	// ----------------------------------------------------------
```

Comment:
Do we do this weird formatting with comments in other parts of the codebase, too?

Reply:
At least in some parts. Movement code had this to begin with, and I removed some of it. Can remove it again, but for a short time it helped me to better see the structure here.

Anyway, the more relevant question here is: Do we still need the if (status ...) return; statements? I think not, because the callbacks cannot remove the object anymore!